schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals(
     date_of_birth        date, 
     escape_attempts     INT, 
     neutered            boolean, 
-    weight_kg           DEC
+    weight_kg           DEC,
+    species             VARCHAR(50)
```

Comment:
It's important that you `ALTER` the `animals` table to add the species column rather than just adding it as a new column. Please make the necessary changes to your code so that your project meets the requirements 📝

You could try using something like:

    ALTER TABLE animals ADD species varchar(255);